Just added a simple logger

pr13260 opened the pull request:
People lazy like me don't like to watch in channel why not send a MSG in inbox for once??

pr13260 committed:
Just added a simple logger

People lazy like me don't like to watch in channel why not send a MSG in inbox for once??

jainamoswal reviewed (changes requested):
First of all, thanks for the PR. ❤️ 

1. The messages are being sent in a loop. So incase there are 5 chats where I need to edit the message (bot status message), this action will send message 5 times. 
> Keep this code outside of `for` loop 

2. You are sending this update to "Saved Messages", still user won't get a **notification** and one needs to keep checking in **saved messages**. So it's same if you edit **channels** or send messages in `"me"`.

pr13260 committed:
That's for updates 

Here are changes requested by you 
•Kept out of loop
•There are scenarios where message to edit is too old and people like editing the same old / first message 
•This is currently sending to saved msgs still users can see faster than scrolling in channel old msgs,  it would also be better if it sent to specific channel/group {That's for future and development things } 

Btw it's a cool project!

jainamoswal reviewed (commented):
Any updates ??

jainamoswal commented:
Assuming this Pull request as stale and closing it fn.
